db/app.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection");
+const express = require("express");
+const app = express();
+const { getCategories } = require('../db/controller')
+
+app.use(express.json());
+
```

Comment:
Make sure you're only adding code that you're using until you need it for post/patch.